tests/Struct/FindologicProductTest.php

```diff
@@ -363,9 +364,15 @@ public function testProductPrices(?Price $price, ?string $exception): void
 
     public function testProduct(): void
     {
-        $productEntity = $this->createTestProduct();
+        $productId = '29d554327a16fd51350688cfa9930b29';
```

Comment:
Where is this ID coming from? How can you ensure that this ID is the same product ID that is created for the following product?